Dockerfile

```diff
@@ -1,16 +1,20 @@
-ARG OPEN_ROUTE_SERVICE_VERSION=8.0.1
+ARG OPEN_ROUTE_SERVICE_VERSION=8.2.0
 
-FROM openrouteservice/openrouteservice:v${OPEN_ROUTE_SERVICE_VERSION} AS builder
+FROM --platform=$BUILDPLATFORM openrouteservice/openrouteservice:v${OPEN_ROUTE_SERVICE_VERSION} AS builder
 
-COPY preparation.sh /preparation.sh
-
-RUN wget http://download.geofabrik.de/north-america/us/massachusetts-latest.osm.pbf -O files/data.osm.pbf
+RUN <<EOT
+apk add osmctools --repository=https://dl-cdn.alpinelinux.org/alpine/edge/testing
+wget http://download.geofabrik.de/north-america/us/massachusetts-latest.osm.pbf -O files/data.osm.pbf
+EOT
```

Comment:
question: What's the benefit of running these in the same layer?

Reply:
Not much, since we aren't catching the layers; I could split them back up if you'd prefer.